### PR TITLE
JSONify Error Response & Refactor to add to Header

### PR DESCRIPTION
**Pull request by kdiluca**

### CODE REVIEW

File: valhalla/odin/service.h [hunk 1]
```diff
@@ -3,13 +3,31 @@
 
 #include <boost/property_tree/ptree.hpp>
 
+#include <prime_server/prime_server.hpp>
+#include <prime_server/http_protocol.hpp>
+
+#include <valhalla/baldr/json.h>
```
Comment (kevinkreiser) [inline valhalla/odin/service.h]: this should be in the source only

File: src/odin/service.cc [hunk 2]
```diff
@@ -21,20 +21,46 @@ using namespace prime_server;
...
 namespace {
   const headers_t::value_type CORS{"Access-Control-Allow-Origin", "*"};
+  const headers_t::value_type JSON_MIME{"Content-type", "application/json;charset=utf-8"};
+  const headers_t::value_type JS_MIME{"Content-type", "application/javascript;charset=utf-8"};
+}
+
+namespace valhalla {
+  namespace odin {
+
+    worker_t::result_t odin_worker_t::jsonify_error(uint64_t code, const std::string& status, const std::string& error, http_request_t::info_t& request_info) const {
```
Comment (kevinkreiser) [inline src/odin/service.cc]: dont forget to move this into an anonymous namespace

Comment (kdiluca) [inline src/odin/service.cc]: This is fixed now!  Thanks.

**Comment by kevinkreiser** [comment]
:rocket:

**Pull request merged**